os/services/unit-test/unit-test.h

```diff
@@ -29,86 +29,97 @@
 
 /**
  * \file
- *	A tool for unit testing Contiki software.
+ *	A tool for unit testing Contiki-NG software.
  * \author
- * 	Nicolas Tsiftes <nvt@sics.se>
+ * 	Nicolas Tsiftes <nvt@acm.org>
  */
 
 #ifndef UNIT_TEST_H
 #define UNIT_TEST_H
 
-#include "sys/rtimer.h"
+#include <stdbool.h>
+#include <stdint.h>
 
-typedef enum unit_test_result {
-  unit_test_failure = 0,
-  unit_test_success = 1
-} unit_test_result_t;
+#include <sys/clock.h>
 
 /**
- * The unit_test structure describes the results of a unit test. Each 
+ * The unit_test structure describes the results of a unit test. Each
  * registered unit test statically allocates an object of this type.
  */
 typedef struct unit_test {
   const char * const descr;
   const char * const test_file;
-  unit_test_result_t result;
+  uint32_t assertions;
+  bool result;
```

Comment:
Is it worth calling this variable `passed` or something similar so the code checking it becomes `if(utp->passed) ..`?